R/import_methods.R

```diff
@@ -320,13 +320,23 @@ function(file,
     }
     if (isTRUE(readxl)) {
         requireNamespace("readxl")
-        if ("rows" %in% names(a)) {
-            warning("'rows' argument ignored when readxl = TRUE. Use 'range' instead.")
```

Comment:
Why delete this?

Reply:
The reason is that the code I added a few lines below...
```
if (length(unused)>0) {
          warning("The following arguments were ignored when readxl = TRUE:",
                  "\n", paste(unused, collapse=', '))
        }
```
...will already issue a warning for any invalid ignored arguments, _including but not limited to_ `rows`. So if somebody passes it a `rows` argument, they will get warned twice. 

That being said, I'll be happy to put the `if` statement back in if you want me to.